src/LightStep/Options.cs

```diff
@@ -223,7 +223,9 @@ private static string GetComponentName()
         private static string GetPlatformVersion()
         {
             var version = ".NET Unknown";
-#if NET45
+#if NETSTANDARD2_0
+            version = System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription;
```

Comment:
This should now be usable in both targets, as NET 4.7.2 includes the full API surface area of NET Standard 2.0.

Reply:
right, good catch